### ISSUE TITLE
Incorrect app ID?

### ISSUE THREAD

**Issue by yselkowitz** [flathub/io.github.chris2511.xca#4]
The upstream website for XCA is https://www.hohnstaedt.de/xca/, and the Mac bundle app ID is de.hohnstaedt.xca (see [Info.plist](https://github.com/chris2511/xca/blob/main/misc/Info.plist.in#L18)); shouldn't that be used for the Flatpak as well?

**Comment by marcthe12** [comment]
Yep, the id is incorrect. The upstream has started shipping appstream image with the other id in this commit https://github.com/chris2511/xca/commit/c4d477c0c9fa3c433a11e81c65bb8f4858bb6561. The maintainer should probably request a change of id if it is allowed. This could allow to upstream any file that is linked to app id.

**Comment by ghost** [comment]
Closed, due to EOL. https://github.com/flathub/io.github.chris2511.xca/pull/14
